src/ui/connect-dashboard-ui/src/FamilyHubs.RequestForSupport.Core/ApiClients/OrganisationClientService.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text.Json;
+using FamilyHubs.ReferralService.Shared.Dto;
+
+namespace FamilyHubs.RequestForSupport.Core.ApiClients;
+
+public interface IOrganisationClientService
+{
+    Task<OrganisationDto?> GetOrganisationDtoByIdAsync(long id);
+}
+
+public class OrganisationClientService : ApiService, IOrganisationClientService
+{
+    public OrganisationClientService(HttpClient client) : base(client)
+    {
+    }
+
+    public async Task<OrganisationDto?> GetOrganisationDtoByIdAsync(long id)
+    {
+        var request = new HttpRequestMessage
+        {
+            Method = HttpMethod.Get,
+            RequestUri = new Uri(Client.BaseAddress + $"api/organisations/{id}"),
+        };
+
+        using var response = await Client.SendAsync(request);
+
+        response.EnsureSuccessStatusCode();
+
+        return await JsonSerializer.DeserializeAsync<OrganisationDto>(await response.Content.ReadAsStreamAsync(), options: new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
```

Comment:
Could move await response.Content.ReadAsStreamAsync() out into own var and also add a static readonly for the options as these won't change and we don't need to create it on each call

Reply:
updated :)